src/Orchard.Web/Modules/Orchard.Autoroute/Services/HomeAliasService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Web.Routing;
+using Orchard.Alias;
+using Orchard.ContentManagement;
+
+namespace Orchard.Autoroute.Services {
+    public class HomeAliasService : IHomeAliasService {
+        private readonly IAliasService _aliasService;
+        private readonly IContentManager _contentManager;
+        private const string AliasSource = "Autoroute:Home";
+        private const string HomeAlias = "";
+
+        public HomeAliasService(IAliasService aliasService, IContentManager contentManager) {
+            _aliasService = aliasService;
+            _contentManager = contentManager;
+        }
+
+        public RouteValueDictionary GetHomeRoute() {
+            return _aliasService.Get(HomeAlias);
+        }
+
+        public int? GetHomePageId() {
+            var homePageRoute = GetHomeRoute();
+            var homePageId = 
+                homePageRoute != null 
+                ? homePageRoute.ContainsKey("id") 
+                    ? XmlHelper.Parse<int>((string)homePageRoute["id"]) 
+                    : default(int?) 
+                : default(int?);
+
+            return homePageId;
+        }
+
+        public IContent GetHomePage(VersionOptions version = null) {
+            var homePageId = GetHomePageId();
+            var homePage = homePageId != null ? _contentManager.Get(homePageId.Value, version ?? VersionOptions.Published) : default(IContent);
```

Comment:
That's what I thought too, but when drilling down into the `Get` method, eventually the `options` argument is accessed without any null checks. There is an overload of `Get` that passes in `VersionOptions.Published`, so 'Published' is the correct default.